index.html

```diff
@@ -0,0 +1,165 @@
+                         consequat sunt nostrud amet.</p>
+            </div>
+            <div class="box-2">
+                <h3>Creating pixel perfect icons in <br>
+                     Figma</h3>
+                     <br>
+                     <div>
+                        <span>12 Feb 2020 </span> &nbsp;
+                        <span class="vl"></span>&nbsp;
+                        <span>Figma, Icon Design</span>
+                    </div>
+                    <br>
+             <p>Amet minim mollit non deserunt ullamco est sit <br>
+                    aliqua dolor do amet sint. Velit officia consequat <br>
+                     duis enim velit mollit. Exercitation veniam <br>
+                     consequat sunt nostrud amet.</p>
+            </div>
+        </div>
+    </div>
+   </section>
+   <!-- third section  -->
+<section>
+    <p class="features">Featured works</p>
+    <div class="board">
+        <div>
+            <img src="img/Rectangle 30.png" alt="#">
+        </div>
+        <div>
+        <h3>Designing Dashboards</h3>
+        <br>
+        <div class="dashboard">
+            <div class="date">2020</div>
+            <div class="event">Dashboard</div>
+        </div>
+        <br>
+        <p class="event-text">Amet minim mollit non deserunt ullamco est sit aliqua dolor do amet sint. Velit <br>
+             officia consequat duis enim velit mollit. Exercitation veniam <br>
+              consequat sunt nostrud amet.</p>
+    </div>
+    </div>
+</section>
+<center><hr></center>
+<section>
+    <div class="board">
+        <div>
+            <img src="img/Rectangle 32.png" alt="#">
+        </div>
+        <div>
+        <h3>Vibrant Portraits of 2020</h3>
+        <br>
+        <div class="dashboard">
+            <div class="date">2018</div>
+            <div class="event">illustration</div>
+        </div>
+        <br>
+        <p class="event-text">Amet minim mollit non deserunt ullamco est sit aliqua dolor do amet sint. Velit <br>
+             officia consequat duis enim velit mollit. Exercitation veniam <br>
+              consequat sunt nostrud amet.</p>
+    </div>
+    </div>
+</section>
+<center><hr></center>
+<section>
+    <div class="board">
+        <div>
+            <img src="img/Rectangle 34.png" alt="#">
+        </div>
+        <div>
+        <h3>36 Days of Malayalam type</h3>
+        <br>
+        <div class="dashboard">
+            <div class="date">2018</div>
+            <div class="event">Typography</div>
+        </div>
+        <br>
+        <p class="event-text">Amet minim mollit non deserunt ullamco est sit aliqua dolor do amet sint. Velit <br>
+             officia consequat duis enim velit mollit. Exercitation veniam <br>
+              consequat sunt nostrud amet.</p>
+    </div>
+    </div>
+</section>
+<center><hr></center>
+<!-- footer area  -->
+<footer>
+    <div class="footer">
+    <div>
+<img src="img/fb.png" alt="#">
```

Comment:
Add <a></a> tags to <img> as links since each social media icon is a link to be clicked.